app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_user!
+
+      def create
+        rent = Rent.create(rent_params)
+        render json: rent
+      end
+
+      def index
+        render json: Rent.all, each_serializer: RentSerializer
```

Comment:
I do not think you need to call the each_serializer there, if it the default one.

Reply:
You forgot pagination ;)

Reply:
For some weird reason (probably versioning) I HAVE TO explicitly specify the serializer for collections of data